app/src/main/java/com/x8bit/bitwarden/ui/platform/feature/vaultunlockednavbar/VaultUnlockedNavBarViewModel.kt

```diff
@@ -168,23 +210,42 @@ sealed class VaultUnlockedNavBarAction {
  * Models events for the bottom tab of the vault unlocked portion of the app.
  */
 sealed class VaultUnlockedNavBarEvent {
+    /**
+     * Whether to return to the root of the subgraph or not.
+     */
+    abstract val returnToSubgraphRoot: Boolean
```

Comment:
So I feel like this applies to all the navigation events but wouldn't technically apply if we ever had to add a different kind of event. This is probably fine for now but it's something to be aware of.